docs/en/01_configuration.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Configuration
+summary: Configuration options, including enabling extraction for DataObjects, managing cached content length, swapping cache backends, and configuring PDF text extraction
+icon: file-text
+---
+
+# Configuration
```

Comment:
Anyone searching for docs for the "configuration" API is gonna have a rough time lol